src/linter_options.h

```diff
@@ -63,45 +66,55 @@ class LinterOptions {
   // INCREASING order.
   void Enable(ErrorCode code, int position);
 
+  // Adds a single parser output to parset_output_
+  void AddParserOutput(ParserOutput *output);
+
+  // ---------------------------------- GETTER/SETTER functions
+
+  // Getter for parser_output_
```

Comment:
Erased extra comments. I also think it would be better without comments, but there is a rule that every function should have an explanation comment, So, one approval to erase these comments is all I need :)